src/cloud/docker/docker-config-blackfire-io.md

```diff
@@ -0,0 +1,109 @@
+---
+group: cloud-guide
+title: Add Blackfire.io to Docker
+functional_areas:
+- Cloud
+- Setup
+- Configuration
+
+---
+
+You can add Blackfire.io to your {{site.data.var.mcd-prod}} environment to complete automated performance testing.
```

Comment:
```suggestion
You can add Blackfire.io to your {{site.data.var.mcd-prod}} environment to get completely automated performance testing.
```

Reply:
"get" is too ambiguous. If you are suggesting that this needs improvement, then I would understand this more clearly:
```suggestion
You can add Blackfire.io to your {{site.data.var.mcd-prod}} environment to fully automate performance testing.
```